arci/src/clients/joint_trajectory_clients_container.rs

```diff
@@ -207,4 +207,101 @@ mod tests {
             "[\"part1\", \"high\", \"part2\", \"low\", \"part3\", \"high\", \"part4\", \"middle\"]"
         );
     }
+
+    #[test]
+    fn test_container_current_pos_ok() {
+        use super::*;
+        #[derive(Debug, Clone)]
+        struct Dummy {
+            name: Vec<String>,
+            pos: Vec<f64>,
+        }
+        impl JointTrajectoryClient for Dummy {
+            fn joint_names(&self) -> &[String] {
+                &self.name
+            }
+            fn current_joint_positions(&self) -> Result<Vec<f64>, Error> {
+                Ok(self.pos.clone())
+            }
+            fn send_joint_positions(
+                &self,
+                _positions: Vec<f64>,
+                _duration: std::time::Duration,
+            ) -> Result<WaitFuture, Error> {
+                unimplemented!();
+            }
+            fn send_joint_trajectory(
+                &self,
+                _trajectory: Vec<TrajectoryPoint>,
+            ) -> Result<WaitFuture, Error> {
+                unimplemented!()
+            }
+        }
+        let clients = vec![
+            Dummy {
+                name: vec![String::from("part1"), String::from("high")],
+                pos: vec![1.0_f64, 3.0_f64],
+            },
+            Dummy {
+                name: vec![String::from("part2"), String::from("low")],
+                pos: vec![2.2_f64, 4.0_f64],
+            },
+        ];
+
+        let container = JointTrajectoryClientsContainer::new(clients.clone());
+        let current = container.current_joint_positions();
+
+        let some = current.unwrap();
+        assert_eq!(format!("{:?}", some), "[1.0, 3.0, 2.2, 4.0]");
```

Comment:
Could you use `assert_approx_eq` to compare the values?

```rust
assert_approx_eq(some[0], 1.0);
assert_approx_eq(some[1], 3.0);
assert_approx_eq(some[2], 2.2);
assert_approx_eq(some[3], 4.0);
```

Reply:
Apply using approx